### DISCUSSION TITLE
Is there a way to programatically reload a plugin?

### DISCUSSION THREAD

**Discussion by ThisIsQasim** [koush/scrypted#1457] [Q&A]
I have a few sensors that rely on MQTT and sometimes after a crash the MQTT server isn't up before scrypted starts so I was thinking if there was a way to programatically reload a plugin preferably using a signal.

**Comment by skynet01** [comment]
Not sure how to reload the plugin with API etc (or if that even exists) , but you can try to use Automations within Scrypted to reload Scrypted plugins or add a waiting period etc. You can also just reload the whole Scrypted container if all else fails :)
